<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigureGeneral</class>
 <widget class="QWidget" name="ConfigureGeneral">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>407</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="HorizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="VerticalLayout">
     <item>
      <widget class="QGroupBox" name="GeneralGroupBox">
       <property name="title">
        <string>General</string>
       </property>
       <layout class="QHBoxLayout" name="GeneralHorizontalLayout">
        <item>
         <layout class="QVBoxLayout" name="GeneralVerticalLayout">
          <item>
           <widget class="QCheckBox" name="toggle_deepscan">
            <property name="text">
             <string>Search sub-directories for games</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="toggle_check_exit">
            <property name="text">
             <string>Confirm exit while emulation is running</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="toggle_user_on_boot">
            <property name="text">
             <string>Prompt for user on game boot</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="PerformanceGroupBox">
       <property name="title">
        <string>Performance</string>
       </property>
       <layout class="QHBoxLayout" name="PerformanceHorizontalLayout">
        <item>
         <layout class="QVBoxLayout" name="PerformanceVerticalLayout">
          <item>
           <widget class="QCheckBox" name="use_cpu_jit">
            <property name="text">
             <string>Enable CPU JIT</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="EmulationGroupBox">
       <property name="title">
        <string>Emulation</string>
       </property>
       <layout class="QHBoxLayout" name="EmulationHorizontalLayout">
        <item>
         <layout class="QVBoxLayout" name="EmulationVerticalLayout">
          <item>
           <widget class="QCheckBox" name="enable_nfc">
            <property name="text">
             <string>Enable NFC</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="theme_group_box">
       <property name="title">
        <string>Theme</string>
       </property>
       <layout class="QHBoxLayout" name="theme_qhbox_layout">
        <item>
         <layout class="QVBoxLayout" name="theme_qvbox_layout">
          <item>
           <layout class="QHBoxLayout" name="theme_qhbox_layout_2">
            <item>
             <widget class="QLabel" name="theme_label">
              <property name="text">
               <string>Theme:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="theme_combobox"/>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="HotKeysGroupBox">
       <property name="title">
        <string>Hotkeys</string>
       </property>
       <layout class="QHBoxLayout" name="HotKeysHorizontalLayout">
        <item>
         <layout class="QVBoxLayout" name="HotKeysVerticalLayout">
          <item>
           <widget class="GHotkeysDialog" name="widget" native="true"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GHotkeysDialog</class>
   <extends>QWidget</extends>
   <header>hotkeys.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
